<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>277</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Reference Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>35</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name: </string>
    </property>
   </widget>
   <widget class="QLabel" name="huID">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>35</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HU ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="password">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLabel" name="createAccount">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create Account</string>
    </property>
   </widget>
   <widget class="QLabel" name="accountType">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Account Type: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="huIDBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="accountTypeDropDown">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Already have an account?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>56</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
